app/code/Magento/ImportExport/Controller/Adminhtml/Import/Download.php

```diff
@@ -36,21 +37,28 @@ class Download extends ImportController
      */
     protected $fileFactory;
 
+    /**
+     * @var \Magento\ImportExport\Model\Import\SampleFileProvider
+     */
+    private $sampleFileProvider;
+
     /**
      * Constructor
      *
      * @param \Magento\Backend\App\Action\Context $context
      * @param \Magento\Framework\App\Response\Http\FileFactory $fileFactory
      * @param \Magento\Framework\Controller\Result\RawFactory $resultRawFactory
      * @param \Magento\Framework\Filesystem\Directory\ReadFactory $readFactory
+     * @param \Magento\ImportExport\Model\Import\SampleFileProvider $sampleFileProvider
```

Comment:
Incorrect order of params in docBlock

Reply:
@ishakhsuvarov fixed

Reply:
In constructor signature `SampleFileProvider` is after `ComponentRegistrar`

Reply:
Done.